### PR TITLE
feat: Added playlist context

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added playlist context
refactored history page, loader and video cards

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -52,10 +52,27 @@ a {
...
 }
 
+.justify-center {
+  justify-items: center;
+}
+
+.as-center {
+  align-self: center;
+}
+
```
Comment (mimansha-swarup) [inline src/App.css:71]: nice work made utility class

**Review by mimansha-swarup — APPROVED** [review]
code looks leam

File: src/components/playlists-list-modal/PlaylistsListModal.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          }}
+        >
+          close
+        </i>
+      </div>
+    </div>
+  );
+}
+
+export { PlaylistsListModal };
```
Comment (manojkumar8400) [inline src/components/playlists-list-modal/PlaylistsListModal.jsx:96]: you can use the default export.

**Review by manojkumar8400 — APPROVED** [review]

File: src/components/playlists-list-modal/PlaylistsListModal.jsx (hunk 2 above)
Comment (manojkumar8400) [reply src/components/playlists-list-modal/PlaylistsListModal.jsx:96]: Else everything is good, good to merge.

**Review by bipin7yadav — APPROVED** [review]
code looks clean you can merge it

File: src/components/playlists-list-modal/PlaylistsListModal.jsx (hunk 2 above)
Comment (Chakravarthi0) [reply src/components/playlists-list-modal/PlaylistsListModal.jsx:96]: Hey Manoj, I am using barrel export thats why i did not use default export here. Thanks for the review

**Pull request merged**